dev-practices/practices.rst

```diff
@@ -0,0 +1,286 @@
+For short-term work, the developer may use the "epic" field, 
+as it does have the advantage that JIRA highlights issues associated to a given epic in scrum boards, 
+but it also has a significant limitation in that only one epic can be assigned to one ticket. 
+Because of that, this field will not be used for general project scheduling.
+
+In general, developers are encouraged to file issues. But please check existing open issues 
+in case the problem or activity has already been addressed,
+ and where appropriate, your manager, before filing.
+
+The normal progression of issue status is “Open” (for a freshly created
+issue), to “In progress”, to “In Review”, to “Reviewed”, to “Done” (when a
+issue has been completed). At any point, an issue can also be set to “Won't
+Fix”.
+
+
+.. _their documentation: https://developer.lsst.io/work/flow.html
+.. _PFS JIRA: https://pfspipe.ipmu.jp/jira
+.. _JIRA project: https://pfspipe.ipmu.jp/jira/secure/BrowseProjects.jspa#all
+
+Workflow
+--------
+
+When the assignee starts work on a given "Open" issue, the issue status should be moved to "In progress". 
+This will help managers and stakeholders know which issues are currently being
+addressed. Then, a corresponding 'ticket branch' should be created. 
+This is a git branch with the name ``tickets/<jira-id>``, 
+where ``<jira-id>`` is the name of the JIRA ticket that is being worked on. 
+For example, when working on ticket "INFRA-26", the branch ``tickets/INFRA-26`` 
+should be created in each of the repositories which are affected. 
+Having this convention helps team members see which branches are related to formal activities, 
+and which JIRA tickets track those activities, as opposed to informal development.
+
+While making your changes, it is highly recommended to include, or update, unit tests or the integration test 
+to demonstrate that the updated code behaves as expected according to the issue description. 
+Please also provide or update documentation where appropriate so that users are aware of the change.
+
+Developers *must* ensure that all unit tests pass before merging code. Reviewers and
+developers are both responsible for checking this.
+
+When work has been completed, please use ``git rebase -i`` to reorganize commits into logical units. 
+Ensure that all unit tests and the integration test pass. 
+Then create a GitHub Pull Request and name one or more reviewers. 
+The pull request will trigger the continuous integration build. Check that the build has passed.
+Then finally set the issue status to “In Review” and assign the same reviewers to the issue as with the pull request.
+
+Please be aware that reviewers may be extremely busy so may not be able to review the issue immediately. 
+However, having a significant amount of code in branches pending to be merged for a long period of time is also problematic, 
+so to remedy such situations the developer should allow a maximum of 7 working days for the reviewer to take action. 
```

Comment:
A week and a half is a significant fraction of the typical sprint.

Reply:
Ok I've reduced this to 5 days which is what you initially proposed I believe. We will see if this is workable or not as we proceed.